index.html

```diff
@@ -646,6 +698,11 @@ <h4>Visually identical text that is not canonically equivalent</h4>
 		</section>
 		    
       </section>
+      <section id="wordBoundary">
+		  <h3>Word boundaries and "whole word" matching</h3>
+		  
+		  <p>Some languages, such as English or Arabic, use spaces between words. Other languages, such as Chinese, Japanese, or Thai, don't. In many non-spacing languages, computing "whole word" matching depends on the ability to determine word boundaries when the boundaries are not themselves encoded into the text.</p>
```

Comment:
In orthographies like Vietnamese (Latin) and Lisu (Fraser), spaces separate syllables rather than words. This may also be something that sub-string matching implementations need to consider.

By the way, although Thai doesn't have word separators, spaces are used to separate phrases, so perhaps the term "non-spacing language" is slightly inaccurate. Maybe consider changing the Thai example to Tibetan?